app/addons/documents/queryoptions/queryoptions.react.jsx

```diff
@@ -30,13 +30,7 @@ var MainFieldsView = React.createClass({
     reduce: React.PropTypes.bool.isRequired,
     toggleReduce: React.PropTypes.func,
     updateGroupLevel: React.PropTypes.func,
-    docURL: React.PropTypes.string
-  },
-
-  getDefaultProps: function () {
```

Comment:
If the docURL is always passed in as a prop, I'd argue keeping `getDefaultProps` results in redundant updates.  No?